symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/stream/util/StreamUtil.java

```diff
@@ -0,0 +1,45 @@
+package com.symphony.bdk.core.service.stream.util;
+
+import org.apiguardian.api.API;
+
+import java.util.Base64;
+
+/**
+ * Helper class providing stream id conversion
+ */
+@API(status = API.Status.EXPERIMENTAL)
+public class StreamUtil {
+
+  private StreamUtil() {
+  }
+
+  /**
+   * Convert the stream id to the corresponding URLSafe encoded stream id
```

Comment:
Please give a conversion example in Javadoc (using `<pre>...</pre>`)

Reply:
done in the method definition for both conversions functions